lego/apps/surveys/views.py

```diff
@@ -68,3 +74,45 @@ def create(self, request, *args, **kwargs):
         return Response(
             SubmissionReadSerializer(serializer.instance).data, status=status.HTTP_201_CREATED
         )
+
+    def validate_answer(self, request, **kwargs):
```

Comment:
Imo `validate_answer` does not describe what this piece of code does.

Something like:
```python
def resolve_answer(self, request, **kwargs):
```
And then a method call on the object.

If there is any logic about what question types can be hidden, it should live inside the model.